hack/boilerplate.go.txt

```diff
@@ -0,0 +1,16 @@
+// This file is part of MinIO Kubernetes Cloud
```

Comment:
Can you remove this change pls. This is unrelated. We can have this as a part of a separate PR if needed

Reply:
To regenerate the clients using `update-code-gen.sh` this file is needed as it provides the license for the generate files.

Reply:
got it